### PR TITLE
feat:cart added 

### PR DESCRIPTION
**Pull request by jatinmore**
Cart Page added

### CODE REVIEW

File: pages/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    float: left;
+    width: 14rem;
+} */
+
+.qty {
+    width: 20%
+}
+
+
+/* .checkout {
```
Comment (teja538) [inline pages/Cart/cart.css:37]: you can remove this comments if not needed

File: styles/style.css [hunk 2]
```diff
@@ -88,6 +88,11 @@
     font-size: 30px;
```
Comment (teja538) [inline styles/style.css:88]: use rems instead of px as you are using rems everywhere

File: pages/Cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/pages/Cart/cart.css">
+    <title>Organix</title>
+</head>
+
+
+<body>
+    <nav class="nav fixed bg-light box-shadow">
+        <a class="brand" href="/index.html">Organix </a>
+        <ul class="navbar-nav">
```
Comment (teja538) [inline pages/Cart/cart.html:16]: you can remove this code if it has no use ... as you have written ul navbar-nav class below

**Review by teja538 — COMMENTED** [review]
looks good...just remove code in comments if not needed

File: pages/Cart/cart.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                    </div>
+               
+                    <hr>
+                    <div class="checkout">
+                        <span>Total</span> <span class="price">550$</span>
+                    </div>
+                    <div class="cart-btn">
+                        <button class="btn dark">Place Order</button>
+                    </div>
+                </div> -->
```
Comment (Manishsharma203) [inline pages/Cart/cart.html:152]: You can remove the commented code

**Pull request merged**